src/content/popup/popup.ts

```diff
@@ -123,9 +125,15 @@ export function renderPopup(
     );
 
     windowElem.dataset.tabSide = options.tabDisplay || 'top';
-  }
 
-  const contentContainer = html('div', { class: 'content' });
+    console.log('Hello1');
+    onHorizontalSwipe(contentContainer, (direction) => {
+      console.log('Hello?');
+      if (options.onSwitchDictionary) {
+        options.onSwitchDictionary(direction === 'left' ? 'prev' : 'next');
+      }
```

Comment:
I think we can just write:

```suggestion
      options.onSwitchDictionary?.(direction === 'left' ? 'prev' : 'next');
```